Config/Tests tvOS.xcconfig

```diff
@@ -0,0 +1,12 @@
+#include "./Tests.xcconfig"
+#include "./tvOS.xcconfig"
+
+SEARCH_PATHS = "$(SRCROOT)/../Vendor/tvOS"
+
+OTHER_CFLAGS = $(inherited) -iframework "$(SRCROOT)/../Vendor/tvOS/OCMock" -iframework "$(SRCROOT)/../Vendor/tvOS/OCHamcrest" -iframework "$(SRCROOT)/../Vendor/tvOS/OHHTTPStubs" -iframework "$(PLATFORM_DIR)/Developer/Library/Frameworks"
+OTHER_LDFLAGS = $(inherited) -framework UIKit -ObjC
+
+LD_RUNPATH_SEARCH_PATHS = @executable_path/Frameworks @loader_path/Frameworks
+
+// OCMock and OCHamcrest have a bunch of warnings so we just diable this only for testing.
```

Comment:
The typo is fixed.
For the second part, we need to fix all warnings from test frameworks by changing their code. This should be done prior to enable the flag so I don't think this PR can address the feedback.
In addition to that, fixing warnings would take time so we should take care of it in a separate task.